lib/inputs/dovetail-impressions.js

```diff
@@ -85,6 +84,25 @@ module.exports = class DovetailImpressions {
         cause:            info.cause
       }
     };
+
+    if (imp.vast) {
```

Comment:
doing all the vast additional data conditionally, jic some part of this data is missing